rest-server/src/components/auth/authController.js

```diff
@@ -41,6 +41,10 @@ export const loginController = async (req, res) => {
     const { id, email } = rows[0];
     console.log('loginController - successfully retrieved data ', rows[0]);
     const token = await genToken(id, email);
+
+    // res.cookie('jwt', token, cookieOptions);
+    // return res.status(200).append('Successful Auth', JSON.stringify(token)).send(rows[0]);
+    res.set('Set-Cookie', `jwt=${token}; Expires=${cookieOptions.maxAge}; HttpOnly`);
```

Comment:
@DChan0319 Do we need both res.set('Set-Cookie') and res.set('jwt')?

but I'm not sure the header 'Set-Cookie' is exposed. We explicitly expose the 'jwt' header on line 49.